index.js

```diff
@@ -1,6 +1,9 @@
 "use strict";
 
-module.exports = function enforceHTTPS() {
+module.exports = function enforceHTTPS(opts = {}) {
+	if (opts.port && opts.port !== 80) {
```

Comment:
`opts?.port !== 80`

Reply:
This line actually checks to see if the port option is set AND if the port number is not port 80. 

What you suggest is not a replacement for that logic.

Reply:
Again, you're clueless. Nevermind.